packages/grammy-nestjs/src/utils/create-bot-factory.util.ts

```diff
@@ -7,6 +7,8 @@ const log = debug('nestjs-grammy:create-bot-factory.util')
 export async function createBotFactory(options: GrammyModuleOptions): Promise<Bot<any>> {
   const bot = new Bot<any>(options.token, options.options)
```

Comment:
Would this be the right time to introduce a type parameter here?
```suggestion
export async function createBotFactory<C extends Context>(options: GrammyModuleOptions<C>): Promise<Bot<C>> {
  const bot = new Bot<C>(options.token, options.options)
```

Reply:
That makes sense to me - the bot does inherit the context type. Would allow for more tight type-definitions. Will add with with this PR - thanks github for making that easy. :)